lecsum-fe/services/api_client.py

```
@@ -97,3 +97,27 @@ def grade_quiz(self, quiz_set_id: int, quiz_id_list: list[int], user_answer_list
+        response = requests.get(f"{self.base_url}/api/quizzes/quiz-sets", params=params)
+        response.raise_for_status()
+        return response.json()
+
+    def get_attempts(self, quiz_set_id: Optional[int] = None, limit: int = 10, offset: int = 0) -> Dict[str, Any]:
+        """응시 기록 목록 조회"""
+        params = {
+            "quiz_set_id": quiz_set_id,
+            "limit": limit,
+            "offset": offset
+        }
+        response = requests.get(f"{self.base_url}/api/quizzes/attempts", params=params)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)